chains/mitosistestnet/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://testnet.mitosiscan.xyz/api
+    family: routescan
```

Comment:
Not sure about that family name. I think it would be more blockscout or something else

Reply:
Yeah it is routerscan
Chk on the explorer

Reply:
you should make family name to "other" cause yours isn't propose by the CLI